packages/mui-joy/src/SvgIcon/SvgIcon.test.tsx

```diff
@@ -56,11 +63,7 @@ describe('<SvgIcon />', () => {
 
   describe('prop: titleAccess', () => {
     it('should be able to make an icon accessible', () => {
-      const { container, queryByText } = render(
-        <SvgIcon title="Go to link" titleAccess="Network">
-          {path}
-        </SvgIcon>,
-      );
+      const { container, queryByText } = render(<SvgIcon titleAccess="Network">{path}</SvgIcon>);
```

Comment:
why did you remove `title`?

Reply:
`title` prop doesn't seem to exist on `SvgIcon` component. Also, the test header's is 
`describe('prop: titleAccess', () ...` which suggest to me that this is a test about the prop `titleAccess`